### ISSUE TITLE
Error when exit app in Android device after update to version 3.0.0

### ISSUE THREAD

**Issue by lucianomlima** [dpa99c/cordova-diagnostic-plugin#75]
If I open my app and exit, everything works fine.
But If I open, go to chrome's console and use plugin's function like `cordova.plugins.diagnostic.isLocationEnabled(function(enabled){console.log('success:', enabled)}, function(e){console.log('error:', e)})`, when exit shows the "App stop" dialog and in the logcat I get the error below.

Cordova version: 6.1.1
Android Platform version: 5.1.1
Device Android version: 5.1.1
Diagnostic plugin version: 3.0.0
```
05-27 02:20:22.492 2047-2047/com.redebahia.guiacorreio E/AndroidRuntime: FATAL EXCEPTION: main
 Process: com.redebahia.guiacorreio, PID: 2047
 java.lang.RuntimeException: Unable to destroy activity {com.redebahia.guiacorreio/com.redebahia.guiacorreio.MainActivity}: java.lang.IllegalArgumentException: Receiver not registered: cordova.plugins.Diagnostic$2@1de3f942
     at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3791)
     at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:3809)
     at android.app.ActivityThread.access$1400(ActivityThread.java:155)
     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1371)
     at android.os.Handler.dispatchMessage(Handler.java:102)
     at android.os.Looper.loop(Looper.java:135)
     at android.app.ActivityThread.main(ActivityThread.java:5343)
     at java.lang.reflect.Method.invoke(Native Method)
     at java.lang.reflect.Method.invoke(Method.java:372)
     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)
     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)
  Caused by: java.lang.IllegalArgumentException: Receiver not registered: cordova.plugins.Diagnostic$2@1de3f942
     at android.app.LoadedApk.forgetReceiverDispatcher(LoadedApk.java:765)
     at android.app.ContextImpl.unregisterReceiver(ContextImpl.java:1805)
     at android.content.ContextWrapper.unregisterReceiver(ContextWrapper.java:511)
     at cordova.plugins.Diagnostic.onDestroy(Diagnostic.java:232)
     at org.apache.cordova.PluginManager.onDestroy(PluginManager.java:298)
     at org.apache.cordova.CordovaWebViewImpl.handleDestroy(CordovaWebViewImpl.java:480)
     at org.apache.cordova.CordovaActivity.onDestroy(CordovaActivity.java:305)
     at android.app.Activity.performDestroy(Activity.java:6189)
     at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1164)
     at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3778)
     at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:3809) 
     at android.app.ActivityThread.access$1400(ActivityThread.java:155) 
     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1371) 
     at android.os.Handler.dispatchMessage(Handler.java:102) 
     at android.os.Looper.loop(Looper.java:135) 
     at android.app.ActivityThread.main(ActivityThread.java:5343) 
     at java.lang.reflect.Method.invoke(Native Method) 
     at java.lang.reflect.Method.invoke(Method.java:372) 
     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905) 
     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)

```

**Comment by dpa99c** [comment]
Thanks for the detailed error report. I'll investigate, fix and push out a patch release.
Note that v3.0.0 contains some major changes/additions, so I was expecting some issues to arise. Hopefully they'll be spotted/fixed quickly so v3 will become stable.

**Comment by dpa99c** [comment]
Fix published to npm in v3.0.1

**Comment by lucianomlima** [comment]
Thank you!
